src/main/java/dao/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package dao;
```

Comment:
The package declaration should be organized according to the task checklist. The FileWriterImpl seems to be a service-related class, so it should be in the `service.impl` package.